sdk/include/reactos/debug.h

```diff
@@ -66,15 +66,15 @@ RtlAssert(
 
 #ifndef assert
 #if DBG && !defined(NASSERT)
-#define assert(x) if (!(x)) { RtlAssert((PVOID)#x, (PVOID)__RELFILE__, __LINE__, ""); }
+#define assert(x) if (!(x)) { RtlAssert((PVOID)#x, (PVOID)__RELFILE__, __LINE__, NULL); }
```

Comment:
This change is pointless

Reply:
Though I agree current code behavior is identical, I argue that it is the only inconsistent call(s) and that it makes modifying `RtlAssert()` itself (i.e. locally) more complicated.

Reply:
Reverted.